conf/janus.transport.http.jcfg.sample

```diff
@@ -19,6 +20,7 @@ general: {
 	#secure_interface = "eth0"		# Whether we should bind this server to a specific interface only
 	#secure_ip = "192.168.0.1"		# Whether we should bind this server to a specific IP address (v4 or v6) only
 	#acl = "127.,192.168.0."		# Only allow requests coming from this comma separated list of addresses
+	#http_lib_debug = false			# Ask libmicrohttpd to write warning and error messages to stderr (default=false)
```

Comment:
I think the property should be called `mhd_debug`, since it's the library we're debugging.

Reply:
Done!